spec/expression.dd

```diff
@@ -1255,6 +1256,41 @@ $(GNAME CastExpression):
         }
         ----
 
+$(H2 $(LNAME2 throw_expression, Throw Expression))
+
+$(GRAMMAR
+$(GNAME ThrowExpression):
+    $(D throw) $(GLINK AssignExpression) $(D ;)
+)
+
+$(P
+    $(I AssignExpression) is evaluated and must be a `Throwable` reference. The
```

Comment:
```suggestion
    $(I AssignExpression) is evaluated; It's type must be covariant with `Throwable`. The
```
? Not a huge fan of this line. Currently on my phone so above is just a sketch

Reply:
Adapted the line to explicitly mention subclasses.